packages/flutter_tools/templates/plugin/pubspec.yaml.tmpl

```diff
@@ -60,7 +60,7 @@ flutter:
 {{/macos}}
 {{#windows}}
       windows:
-        pluginClass: {{pluginClass}}
+        pluginClass: {{pluginClass}}CApi
```

Comment:
Context: https://github.com/flutter/plugins/pull/4156/files#r669929080

I haven't had any better ideas; anything involving a new key is just really painful to roll out to the whole ecosystem (since it would need to be added to Flutter, then percolate to stable, then plugins using the new key would have to require that version as a minimum).

This makes the name of the C registrar function a bit weird, and it looks a bit weird here, but the file names work out pretty well, and that's what plugin developers will actually be interacting with.

One option is to do this, but then also start laying the groundwork for a new key, with the goal of revisiting this in, say, a year when the key support is widely available.

Reply:
Thanks for the clarification. Agreed on the file naming being more important day to day.